pkgs/top-level/all-packages.nix

```diff
@@ -14593,9 +14593,10 @@ with pkgs;
 
   smlpkg = callPackage ../tools/package-management/smlpkg { };
 
-  solc = callPackage ../development/compilers/solc {
+  inherit (callPackages ../development/compilers/solc {
     boost = boost172;
-  };
+  }) solc_0_8_11 solc_0_8_13;
+  solc = solc_0_8_13;
```

Comment:
Do we really need both versions?

Reply:
Yes I think it would be helpful, especially for tests

Reply:
Can we happily ignore those pinnings? A specific patch version shouldn't really make a difference. Also if some package needs a specific version we can add it only for that package, to prevent adding to many versions in top-level.